common/packages.hcl

```diff
@@ -128,6 +128,16 @@ target "_pkg-ubuntu2204" {
   }
 }
 
+target "_pkg-ubuntu2210" {
+  args = {
+    PKG_RELEASE = "ubuntu2210"
+    PKG_TYPE = "deb"
+    PKG_DISTRO = "ubuntu"
+    PKG_SUITE = "kinetic"
+    PKG_BASE_IMAGE = "ubuntu:kinetic"
```

Comment:
I always have to look up the code-names; even wondering if we should start using the CalVer versions for the image references instead (`ubuntu:22.10`). Either way; not for this PR (if we do, we should do it for all of them)

Reply:
Agree yes, I just followed `docker-ce-packaging` scheme.

Reply:
Yes, we do a bit of a mix between "codename" and "version" (packages are named after the codename). I usually know the names for the LTS releases, but always forget the non-LTS ones 😂